### PR TITLE
JDBC for all models added

### PR DESCRIPTION
**Pull request by p1q**

### CODE REVIEW

File: src/main/java/internetshop/dao/jdbc/BucketDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+
+@Dao
+public class BucketDaoJdbcImpl extends AbstractDao<Bucket> implements BucketDao {
+    private static final Logger LOGGER = Logger.getLogger(UserDaoJdbcImpl.class);
```
Comment (Sofasmile) [inline src/main/java/internetshop/dao/jdbc/BucketDaoJdbcImpl.java]:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(BucketDaoJdbcImpl.class);
```

File: src/main/java/internetshop/dao/jdbc/OrderDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.log4j.Logger;
+
+@Dao
+public class OrderDaoJdbcImpl extends AbstractDao<Item> implements OrderDao {
+    private static final Logger LOGGER = Logger.getLogger(UserDaoJdbcImpl.class);
```
Comment (Sofasmile) [inline src/main/java/internetshop/dao/jdbc/OrderDaoJdbcImpl.java]:
```suggestion
    private static final Logger LOGGER = Logger.getLogger(OrderDaoJdbcImpl.class);
```

File: src/main/java/internetshop/dao/jdbc/BucketDaoJdbcImpl.java (hunk 1 above)
Comment (p1q) [reply src/main/java/internetshop/dao/jdbc/BucketDaoJdbcImpl.java]: Done, thanks.

File: src/main/java/internetshop/dao/jdbc/OrderDaoJdbcImpl.java (hunk 2 above)
Comment (p1q) [reply src/main/java/internetshop/dao/jdbc/OrderDaoJdbcImpl.java]: Done, thanks.

**Pull request merged**